### DISCUSSION TITLE
Why doesn't github show this repository as a fork of rocksdb?

### DISCUSSION THREAD

**Discussion by MBkkt** [speedb-io/speedb#271] [Q&A]
This makes it quite inconvenient (but still possible) to see the difference between rocksdb and this repository

**Comment by AmnonHanuhov** [comment]
@MBkkt Because we aren't a fork of rocksdb (due to historical reasons). Originally, speedb started as a project on gitlab and the way the repository was built there was with rocksdb simply being cloned into speedb. When we transitioned to GitHub, the structure of the repo remained the same, that is, not being a fork of rocksdb.

**Reply by MBkkt** [reply]
> When we transitioned to GitHub, the structure of the repo remained the same, that is, not being a fork of rocksdb.

Readme:

> This project is maintained by Speedb and is forked from Rocksdb, developed by Facebook.

Btw I think it also was possible when you uploaded repo to GitHub, anyway thanks for answer

**Comment by mrambacher** [comment]
As an aside, I do not think it is possible to create a fork of a fork in GitHub currently (it was not the last I tried).  Without this "fork of a fork", development in Speedb could get more awkward.

I have solved this problem for myself by adding different remotes for RocksDB and Speedb.

**Comment by MBkkt** [comment]
It's definitely possible to create a fork of a fork, and I pretty sure it was possible at least starts from 2019

Example:
https://github.com/YACLib/cppcoro-for-benchmark

**Comment by MBkkt** [comment]
https://github.com/guoyu07/rocksdb-1 example for rocksdb from 2016
